lib/Core/Executor.h

```diff
@@ -101,7 +101,24 @@ class Executor : public Interpreter {
 
   typedef std::pair<ExecutionState*,ExecutionState*> StatePair;
 
+  enum TerminateReason {
+    Abort,
+    Assert,
+    Exec,
+    External,
+    Free,
+    Model,
+    Overflow,
+    Ptr,
+    ReadOnly,
+    ReportError,
+    User,
+    Unhandled,
```

Comment:
On 08/04/2016, 02:26 PM, Andrea Mattavelli wrote:
> The last element in an Enum should be without the ending comma ','

Why? In the kernel we always put the comma there so that there is no
need to change two lines when augmenting the enum.

-- 
js